Source/AwsCommonRuntimeKit/http/HTTPStream.swift

```diff
@@ -41,7 +41,12 @@ public class HTTPStream {
             throw CommonRunTimeError.crtError(.makeFromLastError())
         }
     }
-
+    
+    /// This method must be overriden in each subclass.
```

Comment:
is this how interface functions are supposed to be documented? doesn't seem very useful to the user

Should this document how you have to set up the connection with "use manual data writes" or "transfer-encoding: chunked" depending on whether it's HTTP/1 or HTTP/2?

Reply:
I have updated the documentation to link to the H1 and H2 documentation.